src/hooks/Meta.py

```diff
@@ -0,0 +1,18 @@
+# Object classes from AP core, to represent an entire MultiWorld and this individual World that's part of it
+from worlds.AutoWorld import WebWorld
+from typing import Optional
+from BaseClasses import Tutorial
+from ..Data import game_table
+
+# Nothing in the world.py hook is executed by the webhost so any logic you want to do need to be done here
+
+# Return a string containing a custom world description
+# for more info check https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/world%20api.md#docstrings
+def hook_set_world_description(base_doc: str) -> str:
+    return base_doc
+
+
+# this hooks allows you to modify the Webworld that get shown on the WebHost when hosted locally
+# for more info check https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/world%20api.md#webworld-class
+def hook_set_world_webworld(web: WebWorld) -> WebWorld:
+    return web
```

Comment:
Both these hooks feel like they should be in Data.

Reply:
I think this feels over-engineered a bit. The other json files just have a before processed hook, and that makes sense. An after load file hook in Data.py for meta seems good, too, for consistency. Beyond that, I think we can expect people to construct the contents like they would have to with other files. Probably not as much need for individual hooks to target portions of the JSON data.

In the general sense, I see hooks as giving world creators programmatic control over the data broadly, and as a way for them to hook into individual generation steps. Beyond that, in most cases, we're probably getting too granular. 😃

Reply:
I tried to have only one hooks  but the code thats is executed by webhost doesnt support self since its undefine at that moment otherwise I would've done 1 hook that allow to change all the data in one go,
thats why im pluggin myself into the variable with a function

Reply:
I might've found a way to combine the hooks into one, ill talk about it in discord

Reply:
Oh, I was just talking about a hook like the other ones used with the JSON files. Like this: https://github.com/ManualForArchipelago/Manual/blob/main/src/Locations.py#L6